### PR TITLE
Support for click to open file/folder

### PR DESCRIPTION
**Pull request by prashantsengar**
Updated it for Issue #7

### CODE REVIEW

**Comment by jancborchardt** [comment]
Hey @prashantsengar, cool stuff! Welcome to Keep or Sweep! :)

Could you also provide a screenshot for the change? In #7 you said:

> What if we just add 2 more buttons to open the folder or file?

But it should really be integrated into the existing elements as mentioned in the issue. :) 

> Clicking on preview and filename opens the file in the proper viewer or app for it
> Clicking the path opens the folder where the file is in, ideally with the file highlighted (if that’s at all possible)


(Also just for info – I’m working on a rewrite of Keep or Sweep in Vue.js, and hopefully we will be able to use Electron for the desktop too.)

**Comment by prashantsengar** [comment]
I have done what you said in the Issue #7 
I have not added the buttons :)

**Comment by jancborchardt** [comment]
Ah great – sorry I didn’t try it before, was only mobile. :)

Tried it out now, running into these issues:

When clicking on a filename I get this error:
```
sh: 1: /home/jan/keeporsweep-desktop/images/icon-128.png: Permission denied
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.6/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "keeporsweep.py", line 122, in open_folder
    os.startfile(os.path.realpath(pathToFolder))
AttributeError: module 'os' has no attribute 'startfile'

**Comment by prashantsengar** [comment]
I updated the function so that it works on different OS platforms.

Sorry I didn't see your complete message. Yes it is possible to trigger the file opening action on clicking the preview. Will update it

**Comment by prashantsengar** [comment]
Did you try it out?

**Comment by jancborchardt** [comment]
@prashantsengar sorry I didn’t reply, was traveling for some weeks. Will try it out these days! :)

File: keeporsweep.py [hunk 1]
```diff
@@ -144,6 +176,6 @@ def sweep_element(self):
 app.master.configure(background="white")
 # Center window on the screen
 # https://stackoverflow.com/a/28224382
-root.eval('tk::PlaceWindow %s center' % root.winfo_pathname(root.winfo_id()))
+#root.eval('tk::PlaceWindow %s center' % root.winfo_pathname(root.winfo_id()))
```
Comment (jancborchardt) [inline keeporsweep.py]: @prashantsengar any reason why this is commented out? Does it not work on your operating system? Or did you miss to uncomment it? :)

**Comment by jancborchardt** [comment]
@prashantsengar sorry, still running into problems :\ I’m using Ubuntu 18.04, do you need any other details? Which operating systems do you use?

When clicking on the image preview or name:
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.6/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "keeporsweep.py", line 131, in show_file
    self.open_file(pathToFolder)
  File "keeporsweep.py", line 139, in open_file
    subprocess.call([opener, filename])
NameError: name 'subprocess' is not defined
```

When clicking on the folder path:
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.6/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "keeporsweep.py", line 125, in show_folder
    self.open_file(os.path.realpath(pathToFolder))
  File "keeporsweep.py", line 139, in open_file
    subprocess.call([opener, filename])
NameError: name 'subprocess' is not defined
```

**Comment by prashantsengar** [comment]
I was using Windows so I had commented those lines. Sorry, I forgot to uncomment them.

**Comment by jancborchardt** [comment]
@prashantsengar no worries. :) Do you have any ideas about the errors I’m getting? Maybe just try Ubuntu from a Live USB stick or in a VM to check it out?

**Comment by prashantsengar** [comment]
Yes, the file that I merged earlier did not import the `subprocess` module. I have imported it now, and it is gonna run most probably

**Comment by jancborchardt** [comment]
@prashantsengar really really good stuff, now it works very nicely! :slightly_smiling_face: Very cool enhancement for the desktop app! And welcome to Keep or Sweep :)

**Pull request merged**